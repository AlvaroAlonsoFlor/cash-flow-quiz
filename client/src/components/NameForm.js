import React from 'react';
import { SET_NAME } from '../constants/action-types';

const NameForm = (props) => {

    function handleChange (event) {
        console.log(event.target.value);
        
         window.store.dispatch({type: SET_NAME, payload: event.target.value})
    }

   
    
    return(
            <input type='text' onChange={handleChange}/>
        
    )
}

export default NameForm;