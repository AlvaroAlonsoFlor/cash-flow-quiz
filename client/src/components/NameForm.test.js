import React from 'react';
import { shallow, mount } from 'enzyme';

import NameForm from './NameForm.js'

const mockFunction = jest.fn();

describe('NameForm renders', () => { 
    it('NameForm should render correctly with no props', () => {
        const component = shallow( < NameForm / > );

        expect(component).toMatchSnapshot();
    });
});



