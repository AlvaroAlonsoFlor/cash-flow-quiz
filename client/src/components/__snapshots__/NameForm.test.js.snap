// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameForm renders NameForm should render correctly with no props 1`] = `
<input
  onChange={[Function]}
  type="text"
/>
`;
